storage/persistent-storage/persistent-storage-azure-file.adoc

```diff
@@ -0,0 +1,37 @@
+[id="persistent-storage-using-azure-file"]
+= Persistent storage using Azure File
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-azure-file
+toc::[]
+
+{product-title} supports Microsoft Azure File volumes. You can
+provision your {product-title} cluster with persistent storage using Azure.
+Some familiarity with Kubernetes and Azure is assumed.
+
+The Kubernetes persistent volume framework allows administrators to provision a
+cluster with persistent storage and gives users a way to request those
+resources without having any knowledge of the underlying infrastructure.
+Azure File volumes can be provisioned dynamically.
+
+Persistent volumes are not bound to a single project or namespace; they can be
```

Comment:
Persistent volumes or PersistentVolumes?

s/they can be shared/you can share them

Reply:
I've updated this to use the API type, and adjusted the following sentence.